src/Microsoft.TestPlatform.CrossPlatEngine/Helpers/Interfaces/IProcessHelper.cs

```diff
@@ -47,5 +47,12 @@ internal interface IProcessHelper
         /// <param name="processId">process id</param>
         /// <returns>Name of process</returns>
         string GetProcessName(int processId);
+
+        /// <summary>
+        /// checks if the process has exited, & returns exitcode.
+        /// </summary>
+        /// <param name="process">process</param>
+        /// <returns>process exited</returns>
```

Comment:
False if process has not exited, True otherwise. Set exitCode only if process has exited.